src/main/java/dev/blachut/svelte/lang/SvelteInspectionSuppressor.kt

```diff
@@ -0,0 +1,23 @@
+package dev.blachut.svelte.lang
+
+import com.intellij.codeInspection.InspectionSuppressor
+import com.intellij.codeInspection.SuppressQuickFix
+import com.intellij.psi.PsiElement
+import dev.blachut.svelte.lang.psi.SvelteHtmlFile
+
+class SvelteInspectionSuppressor : InspectionSuppressor {
+    override fun isSuppressedFor(element: PsiElement, toolId: String): Boolean {
+        if (element.containingFile is SvelteHtmlFile) {
+            if (toolId == "UnnecessaryLabelJS") {
+                return element.textMatches("$")
+            }
+            if (toolId == "BadExpressionStatementJS") {
```

Comment:
I didn't see problems from this inspection in JS, is this strictly TS related or did I miss something?

Reply:
it fixes useless warning for ``$: name``